### PR TITLE
Fix for https://github.com/snowplow/snowplow-java-tracker/issues/195

### PR DESCRIPTION
**Pull request by sowieso-fruehling**
added closing of response body

### CODE REVIEW

**Comment by BenFradet** [comment]
Thanks a lot 👍 , I'll have a look tomorrow

**Comment by snowplowcla** [comment]
Thanks for your pull request. Is this your first contribution to a Snowplow open source project? Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

📝 Please visit https://github.com/snowplow/snowplow/wiki/CLA to learn more and sign.

Once you've signed, please reply here (e.g. I signed it!) and we'll verify. Thanks.

File: src/main/java/com/snowplowanalytics/snowplow/tracker/http/OkHttpClientAdapter.java [hunk 1]
```diff
@@ -96,41 +96,70 @@ public Object getHttpClient() {
...
+    }
+
+
+    /**
+     * Closes response body as required by OkHttpClient documentation
+     *
+     * @param response OkHttpClient response
+     */
+    private void closeResponseBody(Response response) {
+        if (response != null && response.body()!=null)
```
Comment (BenFradet) [inline src/main/java/com/snowplowanalytics/snowplow/tracker/http/OkHttpClientAdapter.java]: nit missing spaces in `response.body()!=null`

**Comment by BenFradet** [comment]
@sowieso-fruehling did you end up signing the cla? if so please say so the check passes and I can merge :+1:

**Comment by sowieso-fruehling** [comment]
Hi @BenFradet I've just signed it.

**Comment by snowplowcla** [comment]
Confirmed! @sowieso-fruehling has signed the Individual Contributor License Agreement. Thanks so much

**Comment by BenFradet** [comment]
Merged, thanks!

**Pull request merged**